test_notebooks/time_variable_Deff.py

```diff
@@ -0,0 +1,89 @@
+"""
+A script for calculating Deff based on MSD data from t=0 to a user-selected
+timepoint or range of timepoints (in which case Deff is a geometric mean).
+"""
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+import scipy
+from scipy import stats
+
+from bokeh.plotting import figure, show, gridplot
+import zipfile
+
+
+ALPHA = 1
+
+
+zf = zipfile.ZipFile('brain-diffusion_data.zip')
```

Comment:
Are we including the bits of code that aren't functions into our functions files? I thought that would best be demonstrated solely in the notebooks.